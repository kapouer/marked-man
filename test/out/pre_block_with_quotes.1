T(1)                                                                      T(1)

NNAAMMEE
       tt - test

tteesstt
         [ "$11" ]

                                 January 1979                             T(1)
