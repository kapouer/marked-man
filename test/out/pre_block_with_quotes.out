[4mT[24m(1)                        General Commands Manual                       [4mT[24m(1)

[1mNAME[0m
       [1mt [22m- test

[1mtest[0m
         [ "$11" ]

                                 January 1979                             [4mT[24m(1)
